### PR TITLE
update(ssri): missing method and version bump

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- add `merge` method support
- version 8 bump
- maintainer added

https://github.com/npm/ssri#integrity-merge

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/ssri/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
-// Type definitions for ssri 6.0
+// Type definitions for ssri 8.0
```
Comment (ExE-Boss) [inline types/ssri/index.d.ts:1]:
```suggestion
// Type definitions for ssri 7.1
```

File: types/ssri/ssri-tests.ts [hunk 2]
```diff
@@ -178,3 +179,14 @@ ssri.integrityStream({
...
+// ssri.merge
+const data = fs.readFileSync('data.txt');
+const expectedIntegrity = ssri.parse(fs.readFileSync('integrity.txt', 'utf8'));
+const updatedIntegrity = ssri.fromData(data, { algorithms: ['sha512'] });
+expectedIntegrity.merge(updatedIntegrity); // $ExpectType void
+expectedIntegrity.merge(updatedIntegrity, {}); // $ExpectType void
+// $ExpectType void
+expectedIntegrity.merge(updatedIntegrity, {
+    strict: true,
+});
```
Comment (ExE-Boss) [inline types/ssri/ssri-tests.ts:192]: This can be on a single line:
```suggestion
expectedIntegrity.merge(updatedIntegrity, { strict: true }); // $ExpectType void
```

**Review by ExE-Boss — CHANGES_REQUESTED** [review]
`integrity.merge(…)` was added in **ssri v7.1**:

- <https://github.com/npm/ssri/blob/v7.1.0/README.md#integrity-merge>
- <https://github.com/npm/ssri/blob/latest/CHANGELOG.md#710-2019-10-24>

**Review by ExE-Boss — APPROVED** [review]
[LGTM](https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md#acronyms "Looks good to me")

---

**P.S.:** For future reference, you can commit all suggestions in a batch commit from [the files tab](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45593/files).

**Review by huan086 — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge :kiss:

**Pull request merged**